File: fix.py

with open(r'src\app\[domain]\layout.tsx', 'r', encoding='utf-8') as f:
    content = f.read()
import re
content = re.sub(r'fetch\(.*?\);', 'fetch(`https://www.youtube.com/oembed?url=&format=json`);'.replace('`', ''), content)
with open(r'src\app\[domain]\layout.tsx', 'w', encoding='utf-8') as f:
    f.write(content)

File: fix.js

const fs = require("fs");
const path = "src/app/[domain]/layout.tsx";
let content = fs.readFileSync(path, "utf8");
content = content.replace(/fetch\(.*?\);/g, "fetch(`https://www.youtube.com/oembed?url=${encodeURIComponent(url)}&format=json`);");
fs.writeFileSync(path, content);

